scripts/pfcstat

```diff
@@ -175,15 +212,15 @@ Examples:
     uid = str(os.getuid())
     cnstat_file = uid
 
-    cnstat_dir = "/tmp/pfcstat-" + uid
-    cnstat_fqn_file_rx = cnstat_dir + "/" + cnstat_file + "rx"
-    cnstat_fqn_file_tx = cnstat_dir + "/" + cnstat_file + "tx"
+    cnstat_dir = os.path.join(os.sep, "tmp", "pfcstat-{}".format(uid))
+    cnstat_fqn_file_rx = os.path.join(cnstat_dir, "{}rx".format(cnstat_file))
+    cnstat_fqn_file_tx = os.path.join(cnstat_dir, "{}tx".format(cnstat_file))
 
-    pfcstat = Pfcstat()
+    pfcstat = Pfcstat(args.namespace, args.show)
 
     if delete_all_stats:
```

Comment:
I think pfcstat -c should clear stats on all ports and all namespace and save all ports stats in the cached file.
If not, it might lead to confusing output.
For example. if user does the following sequence
 - pfcstat -c
Since the internal ports stats are not cleared, they will not be reset to zero, 
- pfcstat -s all
External ports stats will start from zero while the Internal ports  will still have the old value. This might lead to confusion.